options/locale/locale_de-DE.ini

```diff
@@ -326,7 +326,7 @@ hi_user_x=Hallo <b>%s</b>,
 
 activate_account=Bitte aktiviere dein Konto
 activate_account.title=%s, bitte aktiviere dein Konto
-activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]!
+activate_account.text_1=Hallo <b>%[1]s</b>, danke für deine Registrierung bei %[2]s!
```

Comment:
This needs to be mirrored to Crowdin.

Reply:
APPROVED - will be in next crowdin update (on midnight)